generators/docker-base.js

```diff
@@ -18,21 +18,18 @@
  */
 const shelljs = require('shelljs');
 const chalk = require('chalk');
-const crypto = require('crypto');
 const dockerUtils = require('./docker-utils');
-/**
- * This is the Generator base class.
- * This provides all the public API methods exposed via the module system.
- * The public API methods can be directly utilized as well using commonJS require.
- *
- * The method signatures in public API should not be changed without a major version change
- */
+const { getBase64Secret } = require('./utils');
+
 module.exports = {
     checkDocker: dockerUtils.checkDocker,
     checkImages,
     generateJwtSecret,
     configureImageNames,
-    setAppsFolderPaths
+    setAppsFolderPaths,
```

Comment:
What do you think of merging all of `docker-base.js` functions into the new `generator-docker-base.js`?